src/omnipool/math.rs

```diff
@@ -321,6 +322,30 @@ pub fn calculate_remove_liquidity_state_changes(
         Balance::zero()
     };
 
+    // Calculate withdrawal fee
+    let price_diff = if oracle_price <= current_price {
+        current_price.saturating_sub(oracle_price)
+    } else {
+        oracle_price.saturating_sub(current_price)
+    };
+
+    let fee = price_diff.checked_div(&oracle_price)?;
+
+    // just a defensive check to ensure that min withdrawal fee is really < 1.
+    // Reason being that clamp panics if min value > max value.
+    let min_fee: FixedU128 = min_withdraw_fee.into();
+    if min_fee > FixedU128::one() {
```

Comment:
if it is should be `really < 1` (code comment) then it should return None if `min_fee >= 1`

Reply:
I'll update the comment to be <=